betty/app/__init__.py

```diff
@@ -35,30 +35,60 @@
     from concurrent.futures import Executor
     from betty.plugin import PluginRepository
     from betty.cache import Cache
-    from collections.abc import AsyncIterator, Callable
+    from collections.abc import AsyncIterator
 
 _T = TypeVar("_T")
 
 
+class _ShutdownHttpClient:
```

Comment:
Finish making the ancestry pickleable first. If we manage that with Dill, we won't need classes like this anymore.